### DISCUSSION TITLE
Error using DeePMD potentials with LAMMPS

### DISCUSSION THREAD

**Discussion by vsa1920** [deepmodeling/deepmd-kit#2517] [Q&A]
I am trying to run a LAMMPS simulation using a trained DeePMD potential. I am getting this error message and am stuck with it:

**INVALID_ARGUMENT: Tensor descrpt_attr/rcut:0, specified in either feed_devices or fetch_devices was not found in the Graph
ERROR: DeePMD-kit Error: TensorFlow Error: INVALID_ARGUMENT: Tensor descrpt_attr/rcut:0, specified in either feed_devices or fetch_devices was not found in the Graph (/project/source/lmp/pair_deepmd.cpp:933)**

Any help would be greatly appreciated! Thanks!

**Comment by njzjz** [comment]
See #2293.
